dcicutils/qa_utils.py

```diff
@@ -430,32 +432,58 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 class MockFileSystem:
     """Extremely low-tech mock file system."""
 
-    def __init__(self, files=None, default_encoding='utf-8'):
+    def __init__(self, files=None, default_encoding='utf-8', auto_mirror_files_for_read=False, do_not_auto_mirror=()):
```

Comment:
I'd prefer `None` vs. `()` here but a minor detail.

Reply:
Yeah, a fair point.  In a later PR I'll fix that.  It was trying to be immutable and to default to a list, which this sort of does. But probably better to be None and have it replaced by a list in the body. In other languages (of course I mean Common Lisp) the initial value of the missing args are evaluated on each call so you can actually default things rationally. A pity Python didn't do this.